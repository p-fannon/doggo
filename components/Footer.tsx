export default function Footer() {
    return (
        <footer className="flex flex-nowrap justify-end justify-self-end min-w-full pr-2 py-1 border-t-2 border-slate-200 dark:border-slate-500 z-50">
            <span className="text-xs font-mono">âš¡ Powered by Dog.CEO & AWS</span>
        </footer>
    )
}