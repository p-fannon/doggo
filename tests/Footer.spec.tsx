import Footer from '../components/Footer'
import { render, screen } from '@testing-library/react'
import '@testing-library/jest-dom'

describe('Footer', () => {
    it('renders successfully', () => {
        const { container } = render(<Footer />)

        expect(container).toMatchSnapshot()
        const footer = screen.getByRole('contentinfo')
        expect(footer).toBeInTheDocument()
        expect(footer).toHaveTextContent('Powered by Dog.CEO & AWS')
    })
})