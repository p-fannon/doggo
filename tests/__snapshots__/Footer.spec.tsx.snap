// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders successfully 1`] = `
<div>
  <footer
    class="z-50 fixed inset-x-0 bottom-0"
  >
    <div
      class="flex flex-nowrap justify-end justify-self-end min-w-full pr-2 py-1 border-t-2 border-slate-200 dark:border-slate-500"
    >
      <p
        class="chakra-text text-xs font-footer css-0"
      >
        âš¡ Powered by Dog.CEO & AWS
      </p>
    </div>
  </footer>
</div>
`;
